### PR TITLE
fix: better variable parsing

### PR DESCRIPTION
**Pull request by kellyjosephprice**
| [![PR App][icn]][demo] | Fix CX-138 |
| :--------------------: | :--------: |

## 🧰 Changes

Updates @readme/variable and @readme/syntax-highlighter to pull in some fixes to rendering variables in code.

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@readme/emojis": "^5.0.0",
-    "@readme/syntax-highlighter": "^12.1.1",
+    "@readme/syntax-highlighter": "^12.1.2",
```
Comment (kellyjosephprice) [inline package.json:32]: Should this be a peerDep?

File: docs/variable-tests.md [hunk 2]
```diff
@@ -12,5 +13,12 @@ hidden: true
 and
 
 ```js
-const xyz = "<<defvar>>";
+const xyz = '<<defvar>>';
+```
+
+```
+should \<< be escaped >>
```
Comment (erunion) [inline docs/variable-tests.md:20]: Do you need tests here for `\n` and `\r` cases or are the tests in `@readme/variable` sufficient?

**Review by erunion — APPROVED** [review]

File: docs/variable-tests.md (hunk 2 above)
Comment (kellyjosephprice) [reply docs/variable-tests.md:20]: Hmm, good call, probably worth it.

Comment (kellyjosephprice) [reply docs/variable-tests.md:20]: Technically, `\n` is checked on the next two lines.

Comment (erunion) [reply docs/variable-tests.md:20]: k. i'm cool to skip it. the tests in the variable package are pretty good as-is.

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.65.1</b>](/readmeio/markdown/releases/tag/v6.65.1)
<br>

**Pull request merged**